Sources/OpenCombine/Publishers/Publishers.ReplaceNil.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  Publishers.ReplaceNil.swift
+//  
+//
+//  Created by Joseph Spadafora on 7/4/19.
+//
+
+import Foundation
+
+// swiftlint:disable generic_type_name
+// We need to disable this linting rule to maintain the same public api that as
+// the original Combine interface
```

Comment:
I believe generic parameter name is not part of an API at all, since there's no way it can be referenced by name at the call site.

If I'm not mistaken, let's use the same name as we already use here:
https://github.com/broadwaylamb/OpenCombine/blob/065b9819342d2e8ec753609cb3d60a34903b94cd/Sources/OpenCombine/Publishers/Publishers.Sequence.swift#L240-L242

Reply:
Can you remove the comment as well?

Reply:
@broadwaylamb Done.  Should be good to go now